src/main/frontend/handler/metadata.cljs

```diff
@@ -47,7 +48,7 @@
                        (vec))]
     (p/let [_ (-> (file-handler/create-pages-metadata-file repo)
                   (p/catch (fn [] nil)))]
-      (let [new-content (pr-str all-pages)]
+      (let [new-content (with-out-str (cljs.pprint/pprint all-pages))]
```

Comment:
inspired with stackoverflow answer [clojure - Pretty print to a string in ClojureScript? - Stack Overflow](https://stackoverflow.com/questions/32107313/pretty-print-to-a-string-in-clojurescript)